### ISSUE TITLE
Empty d.ts files are causing consumer errors due to @internal classes not being emitted

### ISSUE THREAD

**Issue by zakhenry** [pleerock/typeorm#11]
See https://github.com/Microsoft/TypeScript/issues/6022 - this may have actually been fixed, but the npm package needs to be recompiled with a later version that includes that fix? I get the following errors (using `typescript@next)
```
Building ts for [server]
/Users/zak/ubiquits/ubiquits/node_modules/typeorm/query-builder/QueryBuilder.d.ts(1,23): error TS2306: File '/Users/zak/ubiquits/ubiquits/node_modules/typeorm/query-builder/alias/Alias.d.ts' is not a module.
/Users/zak/ubiquits/ubiquits/node_modules/typeorm/repository/Repository.d.ts(4,51): error TS2306: File '/Users/zak/ubiquits/ubiquits/node_modules/typeorm/query-builder/transformer/PlainObjectToNewEntityTransformer.d.ts' is not a module.
/Users/zak/ubiquits/ubiquits/node_modules/typeorm/repository/Repository.d.ts(5,56): error TS2306: File '/Users/zak/ubiquits/ubiquits/node_modules/typeorm/query-builder/transformer/PlainObjectToDatabaseEntityTransformer.d.ts' is not a module.
/Users/zak/ubiquits/ubiquits/node_modules/typeorm/repository/Repository.d.ts(6,47): error TS2306: File '/Users/zak/ubiquits/ubiquits/node_modules/typeorm/persistment/EntityPersistOperationsBuilder.d.ts' is not a module.
/Users/zak/ubiquits/ubiquits/node_modules/typeorm/repository/Repository.d.ts(7,42): error TS2306: File '/Users/zak/ubiquits/ubiquits/node_modules/typeorm/persistment/PersistOperationExecutor.d.ts' is not a module.
[17:55:21] TypeScript: 5 semantic errors
```

On inspection, those `d.ts` files reporting errors are empty, and the source files have only classes with `@internal` jsdoc.

**Comment by pleerock** [comment]
yeap Im aware of these errors. will fix them today

**Comment by zakhenry** [comment]
Cool, if it helps any, [RxJS implemented a workaround](https://github.com/ReactiveX/rxjs/issues/1010#issuecomment-163430493) to suppress the error. Though I'm unsure if it is still actually an error with latest TS

**Comment by pleerock** [comment]
fixed in 0.0.2-alpha.43 f5999cd..9ba0eb9

**Comment by zakhenry** [comment]
👍 confirmed working now
